### DISCUSSION TITLE
Executable File Not Launching After Build in UE 5.3 Compatible Version 4.13

### DISCUSSION THREAD

**Discussion by elidrissifaouzir1987** [Sixze/ALS-Refactored#407] [Q&A]
After creating the build, the .exe file does not launch. I've tried on several blank projects, but I always encounter the same issue. I am using version 4.13, which is compatible with UE 5.3.

**Comment by wmj-cloud** [comment]
![image](https://github.com/Sixze/ALS-Refactored/assets/59240342/6923a42d-1030-49fc-9d66-d0a944b354d5)
move it to public ,build project

**Reply by elidrissifaouzir1987** [reply]
thank you for your help, it's working now

**Reply by ConorDamery** [reply]
Why is this necessary?
